CyReP/RiotAes128.c

```diff
@@ -261,9 +261,4 @@ const char *modes[] = {
     ""
 };
 
-const char **riot_aes_modes(void)
```

Comment:
Are we should this should be removed to fix the compiler warning, or just #ifdef out? Not sure if anyone is using it..
so actually, if you verified no samples use it, I'd agree its fine to just remove.